packages/devtools/lib/src/timeline/timeline_service.dart

```diff
@@ -86,11 +86,11 @@ class TimelineService {
       threadNames.add(name);
 
       // iOS - io.flutter.1.ui, Android - 1.ui, Dream (g3) - io.flutter.ui
-      if (name.endsWith('.ui')) {
+      if (name.contains('.ui')) {
```

Comment:
You may want to include the other known patterns of the name above (it sounds like there one where `.ui` has some content after it).

Reply:
done